frontend/ui/widget/htmlboxwidget.lua

```diff
@@ -79,17 +79,12 @@ function HtmlBoxWidget:_render()
         return
     end
 
-    -- In pdfdocument.lua, color is activated only at the moment of
-    -- rendering and then immediately disabled, for safety with kopt.
-    -- We do the same here.
     Mupdf.color = Screen:isColorEnabled()
```

Comment:
No risk of letting it (a global in Mupdf module) enabled, if for some reason it was false?
Or is this a one-time initialization from Screen:isColorEnabled() that we do here in case it was never done before?